### ISSUE TITLE
Condition for inline edit on rows

### ISSUE THREAD

**Issue by JanStarman** [contributte/datagrid#1010]
Hi, is it possible to set condition for rows to allow or disable inline edit for them?
I couldn't find anything in documentation.

i.e: only rows with value > 50 in column XYZ will have the option for inline edit

**Comment by JanStarman** [comment]
Ok for anyone wondering, I solved this after few hours by setting properties as disabled while setting defaults value for the row.

inside of 
`$this->getInlineEdit()->onSetDefaults[] = static function(Container $container, array $item)`
do this:
`$container->getComponent('name')->setDisabled();`

**Comment by elring** [comment]
How about
```
$grid->allowRowsInlineEdit(function($item) {
        return $item->myColum > 50;
      });
```
?

It is somehow hidden in Row section in [docs](https://github.com/contributte/datagrid/blob/master/.docs/row.md#allow-inline-edit).

**Comment by radimvaculik** [comment]
Yes, this is possible using `setRowCondition()` on the inline edit configuration. Here's an example:

```php
$grid->addInlineEdit()
    ->setRowCondition(function ($item): bool {
        return $item->value > 50;
    });
```

This will only show the inline edit option for rows where the condition returns `true`.

Closing as answered.
